sakip_sabanci_config.rb

```diff
@@ -19,7 +21,7 @@
 record = '/xmlns:record/xmlns:metadata/oai_dc:dc'
 
 # Cho Required
-to_field 'id', extract: extract_xml("#{record}/dc:identifier", NS), transform: transform(strip: true)
+to_field 'id', extract: extract_oai('dc:identifier'), transform: transform(strip: true)
```

Comment:
the `extract:`, `transform:` keys are the `TrajectPlus` way of doing things, but I've deprecated this in `TrajectPlus` because traject 3 can now just do:
```
to_field 'id', extract_oai('dc:identifier'), strip
```

Reply:
Ok taking those out gives me this error: ```Exception: NoMethodError: undefined method `oai' for #<TrajectPlus::Extraction::TransformPipeline:0x00005645ebbdbd70>
    /usr/local/bundle/gems/traject_plus-1.0.0/lib/traject_plus/extraction.rb:27:in `public_send'
```
Is this do to the conflicting namespace or is it something else?

Reply:
Never mind, I understand